src/utils/mergeObjectWithArrays.js

```diff
@@ -1,5 +1,10 @@
 import _ from 'lodash';
 
+/**
+ * When this merge runs into objects, it replaces the original array with the new one,
```

Comment:
This function merges objects, replacing old arrays in its properties with the new one, unlike `_.merge` which combines old and new arrays.